gsa/src/gmp/models/nvt.js

```diff
@@ -148,7 +148,14 @@ class Nvt extends Info {
 
     delete ret.refs;
 
-    ret.severity = parseSeverity(ret.cvss_base);
+    if (isDefined(ret.severities)) {
+      const {severities} = ret;
+      const {severity} = severities;
+      ret.severity = parseSeverity(severity.score / 10);
+      ret.severity_origin = parseText(severity.origin);
```

Comment:
Please use camelCase for variables in GSA
```suggestion
      ret.severityOrigin = parseText(severity.origin);
```